acceptance_tests/features/steps/view_response_status.py

```diff
@@ -0,0 +1,31 @@
+
+from behave import when, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import reporting_unit
+
+
+@then('the "{survey}" "{period}" collection exercise for ru "{ru_ref}" displays a view link')
+def assert_view_response_status_link(_, survey, period, ru_ref):
+    reporting_unit.go_to(ru_ref)
+    reporting_unit.click_data_panel('Bricks')
+    response_table = browser.find_by_name('tbl-ce-for-survey')
+    rows = response_table.find_by_tag('tbody').find_by_tag('tr')
+    for row in rows:
+        if '202001' in row.text:
```

Comment:
Similar to before, the method allows you to pass any survey and period in but regardless of what you pass in, it'll always go to 'Bricks' and look for period 202001. Could these use the params that you pass in instead of being hard-coded?

Reply:
i realised this, using the params in the push i just did